saleor/dashboard/order/forms.py

```diff
@@ -195,7 +195,7 @@ class Meta:
         fields = ['tracking_number']
         labels = {
             'tracking_number': pgettext_lazy(
-                'Shipment tracking number', 'Tracking number')}
+                'Integer number', 'Tracking number')}
```

Comment:
How do we know it's an integer? I believe "Shipment tracking number" was a better context here.